features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/package.json

```diff
@@ -50,6 +50,7 @@
         "lodash.merge": "^4.6.2",
         "material-design-icons": "^3.0.1",
         "material-ui-chip-input": "^1.0.0",
+        "moment": "^2.29.1",
```

Comment:
Shall we use `dayjs` [Link](https://day.js.org/) instead of `moment`, because it's relatively very small